plugins/package-managers/gleam/src/main/kotlin/HexApiClient.kt

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2025 The ORT Project Copyright Holders <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.gleam
+
+import java.net.HttpURLConnection
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+
+import org.apache.logging.log4j.kotlin.logger
+
+import org.ossreviewtoolkit.utils.ort.HttpDownloadError
+import org.ossreviewtoolkit.utils.ort.OkHttpClientHelper
+import org.ossreviewtoolkit.utils.ort.downloadText
+import org.ossreviewtoolkit.utils.ort.okHttpClient
+
+/**
+ * Response model for the Hex.pm package API.
+ * See https://github.com/hexpm/specifications/blob/main/apiary.apib
+ */
+@Serializable
+internal data class HexPackageInfo(
+    val name: String,
+    @SerialName("latest_stable_version")
+    val latestStableVersion: String? = null,
```

Comment:
But here, you might want to instead use `namingStrategy = JsonNamingStrategy.SnakeCase` on the `Json` instance.